init.ts

```diff
@@ -83,13 +83,15 @@ const useVSCode = flags.vscode === null
   ? confirm(USE_VSCODE_MESSAGE)
   : flags.vscode;
 
-await Deno.mkdir(join(directory, "routes", "api"), { recursive: true });
-await Deno.mkdir(join(directory, "islands"), { recursive: true });
-await Deno.mkdir(join(directory, "static"), { recursive: true });
+await Deno.mkdir(join(resolvedDirectory, "routes", "api"), { recursive: true });
+await Deno.mkdir(join(resolvedDirectory, "islands"), { recursive: true });
+await Deno.mkdir(join(resolvedDirectory, "static"), { recursive: true });
 if (useVSCode) {
-  await Deno.mkdir(join(directory, ".vscode"), { recursive: true });
+  await Deno.mkdir(join(resolvedDirectory, ".vscode"), { recursive: true });
+}
+if (useTwind) {
```

Comment:
This re-format came from `deno fmt`. I think the rename of `directory` has caused the original to exceed a column limit?